### DISCUSSION TITLE
Getting a delete directory button on other's owned repository

### DISCUSSION THREAD

**Discussion by aditya-shrivastavv** [community/community#67788] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I don't know if it's the right category to raise this issue, but most likely it's a bug.
When looking at any repository, In the page when the view of a directory is selected, After clicking on the more options button (the button with 3 dots), I am getting an option for **Delete directory**.

![image](https://github.com/community/community/assets/81470938/343dd0e2-a64c-4873-9b16-832e2f6fb080)

After clicking on it nothing happens, just says that an unexpected error occurred.
The delete operation does not take place, since I don't have the permission to edit the directory. But the thing is I should not even get that delete button in the first place. Right?

Note: I am not talking about the forked version of the repo, I mean to talk about the main remote repo which I do not own. And I have checked that this bug is not specific to the repo I have listed here, it is happening in all the repos.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by radar-blip** [comment]
I can see this too. 
And no it shouldn't be an option on others repos.

You could file a bug report if you want to, here: [Bug-Report](https://support.github.com/contact/bug-report)
